Version.props

```diff
@@ -18,7 +18,7 @@
        including taking version numbers 2.X.Y from upstream and updating .W if we have any
        hotfixes to microsoft/git.
     -->
-    <GitPackageVersion>2.20211115.1</GitPackageVersion>
+    <GitPackageVersion>2.20220123.2-pr</GitPackageVersion>

```

Comment:
Thanks for doing this, @derrickstolee! Would you mind sharing where to find the updated PackageVersion so I can add it to the docs?

Reply:
This is a manual process, separate from releasing microsoft/git. Our internal `MAINTAINING-MICROSOFT-GIT.md` document has some details.